cwms-data-api/src/test/java/cwms/cda/data/dto/location/kind/OutletTest.java

```diff
@@ -1,78 +1,93 @@
+/*
+ * MIT License
+ * Copyright (c) 2024 Hydrologic Engineering Center
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
 package cwms.cda.data.dto.location.kind;
 
+
+import cwms.cda.data.dto.CwmsId;
 import cwms.cda.data.dto.CwmsIdTest;
 import cwms.cda.data.dto.Location;
-import cwms.cda.data.dto.CwmsId;
 import cwms.cda.formatters.ContentType;
 import cwms.cda.formatters.Formats;
 import org.apache.commons.io.IOUtils;
 import org.junit.jupiter.api.Test;
-
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 import java.time.ZoneId;
-
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
-class OutletTest
-{
-	private static final String SPK = "SPK";
-	private static final String PROJECT_LOC = "location";
-	private static final String OUTLET_LOC = PROJECT_LOC + "-outlet";
+class OutletTest {
```

Comment:
As noted in PR https://github.com/USACE/cwms-data-api/pull/727, this test should have coverage for base location outlets. Does it have this?

Reply:
this needs to be added as you are making assumptions that the internals of this DTO will never differentiate between base and sub location outlets and both are valid use cases.

Reply:
I added tests for serializing and deserializing with just a base location id of "outlet".  Also added a serialize from file test for base location only.